uat-vocabularies/sample-detail-type.ttl

```diff
@@ -52,3 +52,17 @@ smpdt:core-tray-condition a skos:Concept ;
     skos:inScheme <http://linked.data.gov.au/def/sample-detail-type> ;
     skos:prefLabel "Core Tray Condition"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/sample-detail-type> .
+
+smpdt:radiation-reading a skos:Concept ; 
+skos:definition "Radiation reading collected of each pallet, units recorded in microsievert per hour (μSv/hour) "@en ;
+rdfs:isDefinedBy https://linked.data.gov.au/def/sample-detail-type ; 
+skos:inScheme https://linked.data.gov.au/def/sample-detail-type ; 
+skos:prefLabel "Radiation reading"@en ; 
+skos:topConceptOf https://linked.data.gov.au/def/sample-detail-type .
+
+smpdt:core-tray a skos:Concept ; 
+skos:definition "The range of plastic, cardboard or metal tray which holds samples of drill core on a pallet”@en ;
+rdfs:isDefinedBy https://linked.data.gov.au/def/sample-detail-type ; 
+skos:inScheme https://linked.data.gov.au/def/sample-detail-type ; 
+skos:prefLabel "Core tray"@en ; 
+skos:topConceptOf https://linked.data.gov.au/def/sample-detail-type .
```

Comment:
change https://linked to http://linked for all occurrences in the uat-vocabulary sub-repo

```suggestion
smpdt:radiation-reading a skos:Concept ; 
skos:definition "Radiation reading collected of each pallet, units recorded in microsievert per hour (μSv/hour) "@en ;
rdfs:isDefinedBy http://linked.data.gov.au/def/sample-detail-type ; 
skos:inScheme http://linked.data.gov.au/def/sample-detail-type ; 
skos:prefLabel "Radiation reading"@en ; 
skos:topConceptOf http://linked.data.gov.au/def/sample-detail-type .

smpdt:core-tray a skos:Concept ; 
skos:definition "The range of plastic, cardboard or metal tray which holds samples of drill core on a pallet”@en ;
rdfs:isDefinedBy https://linked.data.gov.au/def/sample-detail-type ; 
skos:inScheme https://linked.data.gov.au/def/sample-detail-type ; 
skos:prefLabel "Core tray"@en ; 
skos:topConceptOf https://linked.data.gov.au/def/sample-detail-type .
```

Reply:
Files good with these amendments.